### PR TITLE
Only fix multiline text if no <tspan>'s present.

### PR DESCRIPTION
**Pull request by paulkaplan**
### Resolves

_What Github issue does this resolve (please include link)?_

Fixes issues with importing SVGs with text that already uses `<tspan>`'s to show multi-line text. The svg renderer currently assumes all SVGs just use bare `<text>firstline\nsecondline</text>`. But I'm trying to make the SVGs produced by scratch 3 to be spec compliant, so am baking that multiline export into there. 

The core problem with using the current method is that calling `.textContent` on a structure like 
```
<text>
    <tspan>hello</tspan>
    <tspan>world</tspan>
</text>
```
produces `helloworld` instead of the desired `hello\nworld`.

Basically: don't overfix SVG files!

### CODE REVIEW

**Review by cwillisf — APPROVED** [review]
👍
Do you think we need to worry about cases like the below? It feels like there might be a more-correct solution to this problem, but if this change covers all the cases that we care about then I'm OK with it.
```xml
<text>
    <tspan>weird</tspan>
    contrived
    case
</text>
```

**Comment by paulkaplan** [comment]
@cwillisf yeah that is a good point. Right now I'm most worried about (a) scratch 2 svgs and (b) svgs coming out of scratch paint, but there are all manner of delinquency out beyond those two.
